pykodi/kodi.py

```diff
@@ -263,7 +265,7 @@ async def media_seek(self, position):
             await self._server.Player.Seek(players[0]["playerid"], {"time": time})
 
     async def play_item(self, item):
-        await self._server.Player.Open(**{"item": item})
+        await self._server.Player.Open(item=item)
```

Comment:
This is actually deliberate. See here: https://github.com/OnFreund/PyKodi/pull/11

Reply:
thanks for the reference, my mistake!